opendevin/runtime/docker/ssh_box.py

```diff
@@ -382,13 +382,17 @@ def __ssh_login(self):
             )
             hostname = self.ssh_hostname
             username = 'opendevin' if self.run_as_devin else 'root'
-            logger.info(
-                f'Connecting to {username}@{hostname} via ssh. '
-                f"If you encounter any issues, you can try `ssh -v -p {self._ssh_port} {username}@{hostname}` with the password '{self._ssh_password}' and report the issue on GitHub. "
-                f"If you started OpenDevin with `docker run`, you should try `ssh -v -p {self._ssh_port} {username}@localhost` with the password '{self._ssh_password} on the host machine (where you started the container)."
-            )
+            logger.info(f'Connecting to {username}@{hostname} via ssh. ')
             self.ssh.login(hostname, username, self._ssh_password, port=self._ssh_port)
+            logger.info('Connected to SSH session')
         except pxssh.ExceptionPxssh as e:
+            if config.persist_sandbox:
+                password_msg = 'with your ssh password'
+            else:
+                password_msg = f"with the password '{self._ssh_password}'"
+            logger.warn(
+                f'Some issues in ssh connection, try `ssh -v -p {self._ssh_port} {username}@{hostname}` with the {password_msg} and report the issue on GitHub.'
+            )
```

Comment:
Can we still display the message (ssh login with this username etc) by default?

I think it can help when there isn't an exception here, too